projects/qgrid-core/body/body.let.js

```diff
@@ -17,6 +18,18 @@ export class BodyLet {
 				}
 			});
 
+		observe(model.mouseChanged)
+			.subscribe(({ state }) => {
+				const { code, status, target } = state;
+				if (this.selectedNode && status === 'down') {
+					TextSelection.clear(this.selectedNode);
+				}
```

Comment:
this,selectedNode = null?

Reply:
we have to clear text selection from element and remove 'q-grid-can-select-text' class. Otherwise user will able to select it by mouse